src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -51,7 +51,11 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'MonoAndroid70' ">
     <Compile Include="Platforms\android\**\*.cs" />
     <Compile Include="Platforms\xamarin-common\**\*.cs" />
-  </ItemGroup>  
+  </ItemGroup>
+  
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
+    <Compile Include="Platforms\netcoreapp1.0\**\*.cs" />
```

Comment:
I'll add the `PropertyGroup`, but I can't add the `NETFX_CORE` define since that define currently backs all usings of the `Windows.UI.*` namespaces.